packages/web/jestSetup.js

```diff
@@ -17,3 +6,23 @@ Math.random = function random() {
 require('jest-fetch-mock').enableMocks()
 
 process.env.TZ = 'UTC'
+
+global.IntersectionObserver = class IntersectionObserver {
+  constructor() {}
+
+  disconnect() {
+    return null
+  }
+
+  observe() {
+    return null
+  }
+
+  takeRecords() {
+    return null
+  }
+
+  unobserve() {
+    return null
+  }
+}
```

Comment:
What is the purpose of this code?

Reply:
It's a mock since interaction observer is not available in test env